crates/sui-types/src/certificate_proof.rs

```diff
@@ -18,6 +18,8 @@ pub enum CertificateProof {
     /// In practice, we will always get 2f+1 (effects cert), but theoretically f+1 is enough to prove
     /// that the transaction is valid.
     QuorumExecuted(EpochId),
+    /// Transaction generated by the system, for example Clock update transaction
+    SystemTransaction(EpochId),
```

Comment:
Is it not possible to just use `Checkpoint` variant?

Reply:
`Checkpoint` requires a checkpoint sequence number, which we do not know at this point yet